content/docs/examples/creds-in-ocmconfig.md

```diff
@@ -0,0 +1,230 @@
+
+### Repositories and Consumers
+
+In the examples below, some configuration is located under `configurations[0]`.`repositories`, and some other under `configurations[0]`.`consumers`. This chapter explains the difference between `repositories` and `consumers`, which is potentially not as obvious as one could think.
+
+In this context, `repository` is a place where credentials can be stored, i.e., it is a credentials repository. For example, Docker's `config.json` can store multiple credentials, and in that sense the file serves as a repository that can store and provide credentials. That is why its location is configured under `repositories`. Other examples of credentials repositories can be the NPM's `.npmrc` file or a HashiCorp Vault instance.
+
+A `consumer` is something the credentials are required for. For example, if you need to configure credentials that are required to log in to an OCI registry, one could say that the registry will be consuming these credentials, i.e., the registry is a credentials consumer. That is why it is configured under `consumers`.
+
+### Reuse Credentials Configured for Docker
+
+This `.ocmconfig` file will tell the OCM CLI to use credentials configuration from Docker's `config.json` file.
+
+```yaml
+type: generic.config.ocm.software/v1
+configurations:
+  - type: credentials.config.ocm.software
+    repositories:
+      - repository:
+          type: DockerConfig/v1
+          dockerConfigFile: "~/.docker/config.json"
+```
+
+### Reuse Credentials Configured for npm
+
+This `.ocmconfig` file will tell OCM CLI to use credentials configuration from npm's `.npmrc` file.
+
+```yaml
+type: generic.config.ocm.software/v1
+configurations:
+  - type: credentials.config.ocm.software
+    repositories:
+      - repository:
+          type: NPMConfig/v1
+          npmrcFile: '~/.npmrc'
+```
+
+### Accessing OCI Registries
+
+#### HTTPS and Path
+
+To access artifacts in `https://ghcr.io/open-component-model`:
+
+* The different parts of the URL have to be specified in separate fields: `scheme`, `hostname`, and `pathprefix`
+* The fields `scheme` and `pathprefix` are optional. If not specified, the OCM CLI will use the credentials for all schemes and paths on that host
+* The `password` is the user's basic authentication password. Some OCI registries allow to generate user access tokens, which can also be used for basic authentication
+
+```yaml
+type: generic.config.ocm.software/v1
+configurations:
+  - type: credentials.config.ocm.software
+    consumers:
+      - identity:
+          type: OCIRegistry
+          scheme: https
+          hostname: ghcr.io
+          pathprefix: open-component-model
+        credentials:
+          - type: Credentials
+            properties:
+              username: some-user
+              password: some-token
+```
+
+#### HTTP, Port Number, Empty Path
+
+To access artifacts in `http://127.0.0.1:5001`:
+
+* Note the quotes around the `port` number

```

Comment:
should be resolved by https://github.com/open-component-model/ocm/pull/927

Reply:
#927 works as expected. Quotes removed.